app/components/AnnounceBanner.tsx

```diff
@@ -46,30 +55,19 @@ export default function AnnounceBanner() {
             className="usa-banner__content usa-accordion__content padding-top-2"
             id="announcement-banner-blue"
           >
-            <div className="grid-row">
-              <div className="mobile-lg:grid-col-4">
-                August 1, 2022 12:00-13:00 UTC
-                <div>(best for Atlantic):</div>
-                <Link rel="external" href="https://bit.ly/3Pt2TH9">
-                  https://bit.ly/3Pt2TH9
-                </Link>
-              </div>
-              <div className="mobile-lg:grid-col-4">
-                August 1, 2022 20:00-21:00 UTC <br /> (best for Pacific):
-                <br />
-                <Link rel="external" href="https://bit.ly/3IT7Qqc">
-                  https://bit.ly/3IT7Qqc
-                </Link>
+            {events && (
+              <div className="grid-row">
+                {events.map((event) => (
+                  <div key={event.link} className="mobile-lg:grid-col-4">
+                    {event.time}
+                    <div>(best for {event.region}):</div>
+                    <Link rel="external" href={event.link}>
+                      {event.link}
+                    </Link>
+                  </div>
+                ))}
```

Comment:
It would be simpler to extract this as another component.

Reply:
Will do!